src/pages/about/how-we-support.tsx

```diff
@@ -10,10 +10,10 @@ const HowWeSupport: NextPage = () => {
   return (
     <>
       <Head>
-        <title>Who We Support | Ethereum Foundation Ecosystem Support Program</title>
+        <title>How We Support | Ethereum Foundation Ecosystem Support Program</title>
```

Comment:
I believe there should be a way to append ` | Ethereum Foundation Ecosystem Support Program` so that we don't have to explicitly add it to every page title. We use `react-helmet` for this on our current sites - could we use this or does Next.js provide something similar out of the box?

Reply:
Uhm yes, good idea. Will investigate, probably they already support that.

Reply:
I've implemented the `PageMetadata` component (similar at the one we have in eth.org) just to abstract the title and description for now. Next doesn't use react helmet hence it doesn't have the `titleTemplate` feature but you can workaround that pretty easily. Take a look at the `PageMetadata` component.